docs/pages/eas-update/updating-your-app.mdx

```diff
@@ -77,7 +77,7 @@ Inside **android/app/src/main/res/values/strings.xml**, you'll see the `expo_run
 
 The `EXPO_UPDATE_URL` value should contain your project's ID.
 
-Inside **Expo.plist**, you'll see the following additions:
+Inside **ios/<your-project>/Supporting/Expo.plist**, you'll see the following additions:
```

Comment:
```suggestion
Inside **ios/project-name/Supporting/Expo.plist**, you'll see the following additions:
```

Let's use rephrase this (so that it is consistent with the docs also fixes the CI error).